sources/host-ctr/cmd/host-ctr/main_test.go

```diff
@@ -373,6 +378,36 @@ func TestFetchECRRef(t *testing.T) {
 			false,
 			"ecr.aws/arn:aws-iso-e:ecr:eu-isoe-west-1:111111111111:repository/bottlerocket/container:1.2.3",
 		},
+		{
+			"Parse special region ap-east-2",
+			"111111111111.dkr.ecr.ap-east-2.amazonaws.com/bottlerocket/container:1.2.3",
+			false,
+			"ecr.aws/arn:aws:ecr:ap-east-2:111111111111:repository/bottlerocket/container:1.2.3",
+		},
+		{
+			"Parse special region ap-southeast-6",
+			"111111111111.dkr.ecr.ap-southeast-6.amazonaws.com/bottlerocket/container:1.2.3",
+			false,
+			"ecr.aws/arn:aws:ecr:ap-southeast-6:111111111111:repository/bottlerocket/container:1.2.3",
+		},
+		{
+			"Parse special region us-northeast-1",
+			"111111111111.dkr.ecr.us-northeast-1.amazonaws.com/bottlerocket/container:1.2.3",
+			false,
+			"ecr.aws/arn:aws:ecr:us-northeast-1:111111111111:repository/bottlerocket/container:1.2.3",
+		},
+		{
+			"Parse special region eusc-de-east-1",
+			"111111111111.dkr.ecr.eusc-de-east-1.cloud.adc-e.uk/bottlerocket/container:1.2.3",
```

Comment:
this endpoint should be:

```suggestion
			"111111111111.dkr.ecr.eusc-de-east-1.amazonaws.eu/bottlerocket/container:1.2.3",
```

Reply:
updated!